### DISCUSSION TITLE
oVirt-engine problem

### DISCUSSION THREAD

**Discussion by Pr0t3sTx** [oVirt/ovirt-site#3154] [Q&A] [answered]
Hello everyone

I took over the virtualization environment from the former network administrator, with oVirt Engine version 4.4.10.5-1.el8.noarch. However, they didn't provide me with all the access credentials except for the root account. Until now, I used LDAP authentication on the web interface and logged in with my AD account. I stopped a virtual machine, and since then, I can no longer log in to the web interface. I don't have the credentials for admin@internal, I can't install packages like virtsh or ovirt-shell, and I don't have access credentials for the database. Essentially, I'm unable to do much at the moment. I have tried all possible options to create a new account with login rights on the internal web, but I haven't been successful. I'm running out of ideas, so I would greatly appreciate some assistance.

**Comment by michalskrivanek** [comment, marked as answer]
look at ovirt-aaa-jdbc-tool 
you can reset/unlock admin@internal account there
